tests/test_liftover.py

```diff
@@ -7,45 +7,45 @@ def test_hg19_to_hg38():
     ch = ChainLifter(Genome.HG19, Genome.HG38)
 
     result = ch.convert_coordinate("chr7", 140439611)
-    assert result is not None
+    assert result
     assert len(result) == 1
```

Comment:
For these, we could just stick with checking the len == 1

```suggestion
    assert len(result) == 1
```